From 56a0dee06c406414f916d02173f3dd2cc82edec3 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 11 Jul 2017 00:21:13 +0800
Subject: [PATCH 2095/5242] MLK-15938-1 pwm: Kconfig: add imx pwm support for
 arm64 platform

commit  a42e2164b746a0a833adc535ef540efa521d5579 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add ARM64 platform support for i.MX PWM driver.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/pwm/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pwm/Kconfig b/drivers/pwm/Kconfig
index 7917055..e8461f2 100644
--- a/drivers/pwm/Kconfig
+++ b/drivers/pwm/Kconfig
@@ -192,7 +192,7 @@ config PWM_IMG
 
 config PWM_IMX
 	tristate "i.MX PWM support"
-	depends on ARCH_MXC
+	depends on ARCH_MXC || ARCH_MXC_ARM64
 	help
 	  Generic PWM framework driver for i.MX.
 
-- 
1.7.9.5

